deno/gateway/v9.ts

```diff
@@ -1280,7 +1280,12 @@ export type GatewayThreadCreateDispatch = GatewayChannelModifyDispatch;
 /**
  * https://discord.com/developers/docs/topics/gateway#thread-create
  */
-export type GatewayThreadCreateDispatchData = GatewayChannelModifyDispatchData;
+export type GatewayThreadCreateDispatchData = GatewayChannelModifyDispatchData & {
```

Comment:
Use interfaces

Reply:
It doesn't let me:
<img width="705" alt="Screen Shot 2022-02-01 at 3 56 29 PM" src="https://user-images.githubusercontent.com/77477100/152050117-d7d805a4-e529-492f-8d17-d61461b342c6.png">